journald/journald.cpp

```diff
@@ -75,6 +83,27 @@ class JournaldLoggerProcess : public Process<JournaldLoggerProcess>
       std::strcpy((char*) entries[i].iov_base, entry.c_str());
     }
 
+    if (flags.destination_type != "journald") {
```

Comment:
I agree that we don't want to add any more options; I'm just sharing a general recommended practice, since you never know what somebody's going to hack onto your module years after you're gone. I feel like it's safer (more future-proof) to match options in the inclusion/exclusion logic to the options you're actually trying to include/exclude, rather than inverting the logic just to save a comparison. I'll let this slide though if you're going to fight it.